alpha/experience-platform-extension/schema-creation.md

```diff
@@ -1,6 +1,6 @@
 # XDM Schema
 
-Experience Data Model (XDM) schemas are a set a rules that defines and validates the customer experience data format in Adobe Experience Platform. XDM schemas are composed of one class and zero or more mixins. For more information on XDM Schemas, see [XDM System overview](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/xdm_system/xdm_system_in_experience_platform.md) and [Basics of schema composition](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/schema_composition/schema_composition.md). For information on creating your own schema, see the tutorial on [Create a schema using the Schema Editor](https://www.adobe.io/apis/experienceplatform/home/tutorials/alltutorials.html#!api-specification/markdown/narrative/tutorials/schema_editor_tutorial/schema_editor_tutorial.md).
+Experience Data Model (XDM) schemas are a set a rules that define and validate the customer experience data format in Adobe Experience Platform. XDM schemas are composed of one class and zero or more mixins. For more information about XDM Schemas, see [XDM System overview](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/xdm_system/xdm_system_in_experience_platform.md) and [Basics of schema composition](https://www.adobe.io/apis/experienceplatform/home/xdm/xdmservices.html#!api-specification/markdown/narrative/technical_overview/schema_registry/schema_composition/schema_composition.md). For information on creating your own schema, see the tutorial on [Create a schema using the Schema Editor](https://www.adobe.io/apis/experienceplatform/home/tutorials/alltutorials.html#!api-specification/markdown/narrative/tutorials/schema_editor_tutorial/schema_editor_tutorial.md).
```

Comment:
- In the 1st sentence, please replace "a set a" with "a set of"
- in the 3rd sentence, please replace "information on" with "information about"